src/Utility/Text.php

```diff
@@ -50,6 +50,13 @@ class Text
         'script',
     ];
 
+    /**
+     * Whether to use I18n functions for translating default error messages
+     *
+     * @var bool
+     */
+    protected static bool $_useI18n;
```

Comment:
```suggestion
    protected static bool $useI18n;
```

We don't use underscore prefix for protected properties anymore.

Reply:
Better without underscores: the Class needs a refactoring...